docs/send-data/opentelemetry-collector/troubleshooting-faq.md

```diff
@@ -246,29 +226,28 @@ See [Accessing the collector's metrics](#accessing-the-collectors-metrics) secti
 
 #### How do I uninstall the Sumo OpenTelemetry Collector?
 
-To uninstall the Collector, you can run the following command:
-
-```bash
-curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-otel-collector/main/scripts/install.sh \
-| sudo -E bash -s -- --uninstall --purge --yes
-```
-
-This will completely remove the Collector from your system, including any associated configuration files.
-
-If you are using a Mac, you can also run the following command to clear the cache:
-
-```bash
-sudo rm -rf /var/cache/otelcol-sumo
-```
-
-This will remove any cached data associated with the Collector.
+Refer to the Uninstall section in the below mentioned docs
+* [Linux](https://help.sumologic.com/docs/send-data/opentelemetry-collector/install-collector-linux/#uninstall)
+* [Windows](https://help.sumologic.com/docs/send-data/opentelemetry-collector/install-collector-windows/#uninstall)
+* [MacOS](https://help.sumologic.com/docs/send-data/opentelemetry-collector/install-collector-macos/#uninstall)
 
 
 ## Data ingestion and forwarding
 
 #### How do I ingest historical data​?
 
-To ingest old or historical data is to use a [Filelog Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) for log collection. By default, the Filelog Receiver starts reading logs from the end of the file. However, this can be changed by adjusting the configuration file to start reading from the beginning of the file by setting `start_at: beginning`.
+To ingest old or historical data use a [Filelog Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) for log collection. By default, the Filelog Receiver starts reading logs from the end of the file. However, this can be changed by adjusting the configuration file to start reading from the beginning of the file by setting `start_at: beginning` in the receiver section. Below is an example
```

Comment:
```suggestion
To ingest old or historical data use a [Filelog Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver) for log collection. By default, the Filelog Receiver starts reading logs from the end of the file. However, this can be changed by adjusting the configuration file to start reading from the beginning of the file by setting `start_at: beginning` in the receiver section. Here is an example:
```